### DISCUSSION TITLE
CDN for PnP v2.x or upper

### DISCUSSION THREAD

**Discussion by ggravey** [SharePoint/sp-dev-docs#7594] [Q&A] [answered]
Hi everybody,

I am optimizing a SharePoint app. Among all optimisations envisioned, I want to put my PnP package in a CDN.
I followed the official documentation : https://pnp.github.io/pnpjs/v1/documentation/deployment/

But Unfortunatly, I was unable to find a CDN version for V2.11.0

Did I miss something ?

Thank you

**Comment by andrewconnell** [comment, marked as answer]
You should probably post that to the PnPJS repo (https://github.com/pnp/pnpjs/), not this one that's more specific to official docs. PnPJS is not part of the official releases from Microsoft.

**Reply by ggravey** [reply]
@andrewconnell thank you for your answer.
I will post that to the right place.
